html/elements/plaintext.json

```diff
@@ -0,0 +1,59 @@
+{
+  "html": {
+    "elements": {
+      "plaintext": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/plaintext",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Up to Firefox 3.6 included, this element implemented the &lt;HTMLSpanElement&gt;. From Firefox 4, it implements the compliant &lt;HTMLElement&gt;."
```

Comment:
Please change this note to be the same note we used for the other cases already.
`"notes": "Before Firefox 4, this element implemented the <code>HTMLSpanElement</code> interface instead of the standard <code>HTMLElement</code> interface.` (same for firefox_android)

Reply:
Fixed.